baklava/README.md

```diff
@@ -11,14 +11,14 @@ It uses ElectronJS as a wrapper for [dogehouse.tv](https://dogehouse.tv) and add
 - Splash screen
 - Localization
 
-## How to run
+## How to reproduce builds
```

Comment:
let this stay as "How to run" as build step is optional. again no changes needed in this file here.